fe/fe-core/src/main/java/com/starrocks/credential/hdfs/HDFSCloudConfigurationProvider.java

```diff
@@ -65,7 +66,8 @@ public CloudConfiguration build(Map<String, String> properties) {
                 getOrDefault(properties, HDFS_USERNAME, HDFS_USERNAME_DEPRECATED),
                 getOrDefault(properties, HDFS_PASSWORD, HDFS_PASSWORD_DEPRECATED),
                 getOrDefault(properties, HDFS_KERBEROS_PRINCIPAL, HDFS_KERBEROS_PRINCIPAL_DEPRECATED),
-                getOrDefault(properties, HADOOP_KERBEROS_KEYTAB, HDFS_KERBEROS_KEYTAB_DEPRECATED),
+                getOrDefault(properties, HADOOP_KERBEROS_KEYTAB, HDFS_KERBEROS_KEYTAB_DEPRECATED,
```

Comment:
keytab and ticket cache are two different concepts.

Reply:
> keytab and ticket cache are two different concepts.

Do you think we need to add a new field about ticket cache?

Reply:
would  be better, so there are several orthogonal ways 
- no auth
- simple
- keytab file/content + principal
- ticket cache